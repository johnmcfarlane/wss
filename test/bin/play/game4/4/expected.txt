 35  6,1  - ARENT
 33  5,1  - RETAN
 30  6,1  - AREN
 30  6,1  - ATEN
 30  7,1  - RANT
 30  7,1  - RENT
 28  6,1  - ANET
 28  7,1  - TAEN
 25  8,1  - ANT
 25  7,1  - EAN
 25  7,1  - RAN
 25  7,1  - REN
 25  7,1  - TAN
 25  7,1  - TEN
 24  6,1  - ARET
 24  6,1  - ERAT
 23  3,1  - ATREN
 23  7,1  - NAE
 23  7,1  - NAT
 23  7,1  - NET
 23  3,1  - RETAN
 20  8,1  - AN
 20  2,1  - ANTRE
 20  8,1  - EN
 20  9,1  - NA
 20  2,1  - TERNA
 19  7,1  - AET
 19  8,1  - AET
 19  7,1  - EAT
 19  8,1  - ETA
 19  9,5  | QINTAR
 19  7,1  - RAT
 19  7,1  - RET
 19  7,1  - TAE
 18  9,1  - EN
 18  9,1  | EON
 18  9,1  | RON
 18  9,1  | TON
 17  3,1  - ANRE
 17  3,1  - ANTE
 17  3,1  - ARNE
 17  3,3  - ATREN
 17  3,1  - ETNA
 17  3,1  - NARE
 17  3,1  - NATE
 17  3,1  - RANE
 17  3,1  - TANE
 15  4,3  - AREN
 15  4,3  - ATEN
 15  9,1  | NOA
 15  9,1  | NOR
 15  9,1  | NOT
 15  4,3  - TAEN
 14  8,1  - AE
 14  3,5  - ARENT
 14  3,1  - ARTE
 14  8,1  - AT
 14  8,1  - ET
 14  9,1  - ET
 14  3,1  - RATE
 14  5,3  - REN
 14  9,1  - TA
 14  3,1  - TARE
 14  5,3  - TEN
 13  6,7  - ANE
 13  4,3  - ANET
 12  6,6  | ANEAR
 12  6,6  | ANTAE
 12  6,6  | ANTAR
 12  6,6  | ANTRA
 12  6,6  | ANTRE
 12  9,1  | AOR
 12  6,6  | AREAN
 12  6,6  | ARENA
 12  6,6  | ARENT
 12  2,5  - ATREN
 12  6,6  | ATREN
 12  6,3  - EN
 12  5,3  - NET
 12  5,1  - RANE
 12  4,5  - RANT
 12  4,5  - RENT
 12  2,5  - RETAN
 12  9,1  | ROE
 12  9,1  | ROT
 12  5,1  - TANE
 12  9,1  | TOA
 12  9,1  | TOE
 12  9,1  | TOR
 12  8,6  | YENTA
 11  6,1  - ANE
 11  5,5  - ANT
 11  2,3  - ANTRE
 11  3,5  - AREN
 11  4,3  - ARET
 11  6,7  - ATE
 11  3,5  - ATEN
 11  6,7  - ETA
 11  5,1  - NARE
 11  5,1  - NATE
 11  3,5  - NEAT
 11  4,5  - NEAT
 11  6,7  - RTE
 11  3,5  - TAEN
 11  3,5  - TEAN
 11  3,5  - TRAN
 10  5,3  - AER
 10  5,3  - AET
 10  3,5  - ANET
 10  4,1  - AREN
 10  3,3  - ARNE
 10  4,1  - ATEN
 10  9,3  - EAU
 10  7,7  - NA
 10  3,3  - NARE
 10  3,3  - NATE
 10  7,7  - NE
 10  3,3  - RANE
 10  5,1  - RATE
 10  5,3  - RET
 10 10,4  | RIDENT
 10  4,1  - TAEN
 10  3,3  - TANE
 10  5,1  - TARE
 10  4,1  - TEAN
 10  4,1  - TRAN
 10  8,6  | YEARN
 10  8,6  | YEN
  9  4,1  - ANET
  9  3,3  - ANRE
  9  2,5  - ANRE
  9  3,3  - ANTE
  9  2,5  - ANTE
  9  1,5  - ANTRE
  9  6,1  - ARE
  9  3,5  - ARET
  9  6,1  - ATE
  9  5,1  - EAN
  9  4,5  - EAN
  9  5,5  - EAT
  9  7,2  | ENTRIA
  9  6,1  - ERA
  9  3,5  - ERAT
  9  6,1  - ETA
  9  5,5  - ETA
  9  4,1  - NEAR
  9  4,1  - NEAT
  9 10,4  | NIDE
  9  9,1  | NO
  9  5,1  - RAN
  9  4,5  - RAN
  9  5,1  - REN
  9  4,5  - REN
  9 10,4  | RIDEN
  9  5,6  | STENAR
  9  5,1  - TAN
  9  4,5  - TAN
  9  5,1  - TEN
  9  4,5  - TEN
  9  1,5  - TERNA
  9  8,6  | YARN
  9  8,6  | YEAN
  9  8,6  | YERN
  8 10,4  | AIDER
  8  6,1  - AN
  8  5,5  - AN
  8  6,7  - AN
  8  4,3  - ANE
  8  4,1  - ARET
  8  2,5  - ARNE
  8  3,3  - ARTE
  8  5,5  | ASTERN
  8 10,1  | EAN
  8  7,2  | EARNIE
  8  6,1  - EN
  8  5,5  - EN
  8  6,7  - EN
  8  5,4  | ENSTAR
  8  6,3  - ER
  8  4,1  - ERAT
  8  6,3  - ET
  8  2,5  - ETNA
  8  7,6  | IRENA
  8  7,1  - NA
  8  6,5  - NA
  8  5,1  - NAE
  8  3,5  - NAE
  8  5,1  - NAR
  8  2,5  - NARE
  8  5,1  - NAT
  8  4,5  - NAT
  8  2,5  - NATE
  8  7,1  - NE
  8 10,4  - NE
  8  5,1  - NEA
  8  3,5  - NEA
  8  4,5  - NEA
  8  5,1  - NET
  8  4,5  - NET
  8 10,4  | NID
  8 10,1  | RAN
  8  2,5  - RANE
  8  3,3  - RATE
  8  8,2  | SAN
  8  8,2  | SEN
  8  5,6  | STANE
  8  5,6  | STERNA
  8  7,7  - TA
  8 10,1  | TAN
  8  2,5  - TANE
  8  3,3  - TARE
  8  7,7  - TE
  8  4,1  - TEAR
  8  8,6  | YARE
  8  8,6  | YATE
  8  8,6  | YEAR
  8  8,6  | YEAT
  8  8,6  | YETA
  7  5,1  - AER
  7  5,1  - AET
  7  4,5  - AET
  7 10,4  | AIDE
  7 10,2  | AN
  7 10,7  - AN
  7  3,5  - ANE
  7  2,5  - ARTE
  7  5,1  - EAR
  7  5,1  - EAT
  7  4,5  - EAT
  7 10,7  - EN
  7  5,3  | ERNST
  7 10,5  | IDANT
  7 10,5  | IDEAN
  7 10,5  | IDENT
  7  7,6  | INERT
  7  7,6  | INTER
  7  7,6  | INTRA
  7  7,6  | IRAN
  7  7,6  | ITEN
  7  4,3  - NAE
  7  5,3  - NE
  7  5,3  | NETS
  7  7,5  | NITER
  7  7,5  | NITRE
  7 10,3  - NU
  7  5,1  - RAT
  7  4,5  - RAT
  7  2,5  - RATE
  7  5,1  - REA
  7  4,5  - REA
  7  7,1  | RENT
  7  5,1  - RET
  7  4,5  - RET
  7  7,5  | RIANT
  7 10,4  | RIDE
  7  5,6  | SANER
  7  5,6  | SEAN
  7  5,6  | SNARE
  7  5,6  | STAN
  7  5,6  | STARN
  7  5,6  | STEAN
  7  5,6  | STERN
  7  5,1  - TAE
  7  5,1  - TAR
  7  2,5  - TARE
  7  5,1  - TEA
  7  4,5  - TEA
  7 10,4  | TIDE
  7  7,5  | TINEA
  7  8,6  | YAR
  7  8,6  | YAT
  7  8,6  | YEA
  7  8,6  | YER
  7  8,6  | YET
  6  6,1  - AE
  6  7,1  - AE
  6 10,4  - AE
  6 10,7  - AE
  6 10,4  | AID
  6  4,1  - ANE
  6  5,3  | ANES
  6  5,4  | ANSER
  6  5,3  | ANTS
  6  6,1  - AR
  6 10,7  - AR
  6  4,3  - ARE
  6  3,5  - ARE
  6  5,3  | AREST
  6  5,5  | ASTER
  6  8,1  | ASTR
  6  5,5  | ASTRE
  6  6,1  - AT
  6  5,5  - AT
  6  6,5  - AT
  6  6,7  - AT
  6 10,7  - AT
  6  4,3  - ATE
  6  3,5  - ATE
  6  7,1  | EEN
  6  9,1  | EO
  6  6,1  - ER
  6 10,7  - ER
  6  3,5  - ERA
  6  7,4  | ERIAN
  6  5,3  | ERNS
  6  6,1  | ERRATA
  6  6,1  - ET
  6  5,5  - ET
  6  6,7  - ET
  6 10,7  - ET
  6  3,5  - ETA
  6 10,5  | IDETA
  6  7,6  | INTA
  6  7,6  | INTR
  6  7,6  | IRATE
  6  4,1  - NAE
  6  4,1  - NEA
  6  7,1  | NEAR
  6  7,1  | NEAT
  6  7,1  | NEER
  6  7,1  | NEET
  6  7,1  | NERE
  6  7,1  | NETE
  6  5,4  | RASEN
  6  6,5  | RATAN
  6  7,1  - RE
  6 10,4  - RE
  6  3,5  - REA
  6  5,3  | REAST
  6  7,1  | REEN
  6  7,4  | REINA
  6  7,1  | REN
  6 10,4  | RID
  6  4,3  - RTE
  6  3,5  - RTE
  6  5,6  | SANE
  6  5,6  | SANT
  6  5,6  | SENA
  6  5,6  | SENT
  6  5,6  | SERTA
  6  5,6  | STARE
  6  5,6  | STRAE
  6  7,1  - TA
  6  6,5  - TA
  6  4,3  - TAE
  6  3,5  - TAE
  6  7,4  | TAIRN
  6  5,3  | TANS
  6  5,3  | TARSE
  6  7,1  - TE
  6 10,4  - TE
  6  3,5  - TEA
  6  7,1  | TEAN
  6  7,1  | TEEN
  6  7,4  | TEIAN
  6  7,1  | TEN
  6  5,3  | TENS
  6  7,1  | TERN
  6 10,4  | TID
  6  9,1  | TO
  6  3,5  - TRA
  6  7,4  | TRINA
  6  7,4  | TRINE
  6  8,1  | TSAR
  5  5,3  - AE
  5  7,5  | AINE
  5  7,5  | AINT
  5  7,5  | AIRN
  5  6,4  | ANAT
  5  6,6  | ANAT
  5  6,6  | ANET
  5  6,6  | ANRE
  5  6,6  | ANTA
  5  6,6  | ANTE
  5  4,1  - ARE
  5  6,1  | AREN
  5  6,6  | AREN
  5  5,3  | ARES
  5  7,4  | ARITE
  5  6,6  | ARNA
  5  6,1  | ARNE
  5  6,6  | ARNE
  5  5,3  | ARTS
  5  4,1  - ATE
  5  6,6  | ATEN
  5  5,3  | ATES
  5  7,2  | EAN
  5  6,5  | EARN
  5  5,3  | EARS
  5  5,3  | EATS
  5  7,2  | EEN
  5  7,2  | EN
  5  4,1  - ERA
  5  5,3  | ERAS
  5  7,4  | ERIN
  5  7,2  | ERN
  5  4,1  - ETA
  5  5,3  | ETAS
  5  7,6  | IARE
  5 10,5  | IDEA
  5  7,6  | ITEA
  5  7,6  | ITER
  5  5,1  - NA
  5  4,5  - NA
  5  7,4  | NAIR
  5  6,5  | NARE
  5  5,4  | NAST
  5  6,5  | NATE
  5  6,5  | NATR
  5  5,1  - NE
  5  4,5  - NE
  5  7,1  | NEA
  5  6,4  | NEAR
  5  6,4  | NEAT
  5  7,1  | NEE
  5  5,4  | NEST
  5  7,1  | NET
  5  9,2  | ON
  5  7,4  | RAIN
  5  6,5  | RANA
  5  6,5  | RANE
  5  6,5  | RANT
  5  5,3  | RATS
  5  5,3  - RE
  5  4,1  - REA
  5  6,2  | REATA
  5  6,4  | REATA
  5  7,1  | REET
  5  7,4  | REIN
  5  5,4  | RESAT
  5  7,1  | RETE
  5  5,3  | RETS
  5  7,5  | RINE
  5  4,1  - RTE
  5  8,2  | SAE
  5  8,2  | SAR
  5  5,6  | SARE
  5  5,6  | SART
  5  8,2  | SAT
  5  5,6  | SATE
  5  8,2  | SEA
  5  5,6  | SEAR
  5  5,6  | SEAT
  5  8,2  | SER
  5  5,6  | SERA
  5  5,6  | SERT
  5  8,2  | SET
  5  5,6  | SETA
  5  8,2  | STA
  5  5,6  | STAR
  5  5,6  | STER
  5  8,2  | STR
  5  5,6  | STRA
  5  5,6  | STRE
  5  4,1  - TAE
  5  6,5  | TAEN
  5  7,4  | TAIN
  5  6,5  | TANA
  5  6,5  | TANE
  5  6,5  | TAREA
  5  6,5  | TARN
  5  5,3  | TARS
  5  5,3  - TE
  5  4,1  - TEA
  5  6,4  | TEAN
  5  7,1  | TEAR
  5  5,3  | TEAS
  5  7,5  | TIEN
  5  7,5  | TINA
  5  7,5  | TINE
  5  4,1  - TRA
  5  6,1  | TRAN
  5  6,4  | TRAN
  5  5,3  | TRES
  5  7,4  | TRIN
  4  5,1  - AE
  4  4,5  - AE
  4  7,1  | AER
  4  7,1  | AET
  4  7,5  | AIN
  4  7,5  | AIRE
  4  7,5  | AIRT
  4  6,4  | ANA
  4  6,6  | ANA
  4  6,6  | ANE
  4  7,4  | ANI
  4  5,4  | ANS
  4  6,6  | ANT
  4  6,6  | AREA
  4  6,1  | ARET
  4  6,6  | ARET
  4  6,1  | ARN
  4  6,6  | ARN
  4  5,4  | ARSE
  4  6,1  | ARTE
  4  6,6  | ARTE
  4  5,5  | ASTR
  4  6,4  | ATAR
  4  6,6  | ATAR
  4  6,5  | EAN
  4  7,2  | EAR
  4  5,4  | EAST
  4  7,2  | EAT
  4  7,1  | EER
  4  7,2  | EER
  4  5,4  | ENS
  4  7,2  | ERA
  4  6,1  | ERAT
  4  6,4  | ERAT
  4  7,2  | ERE
  4  7,4  | ERIA
  4  6,1  | ERN
  4  5,4  | ERST
  4  7,2  | ETA
  4  7,6  | IAN
  4 10,5  | IDA
  4 10,5  | IDE
  4  7,6  | INT
  4  6,4  | NAA
  4  6,5  | NAA
  4  6,5  | NAE
  4  6,5  | NAR
  4  6,5  | NAT
  4  6,4  | NEA
  4  7,4  | NEI
  4  7,5  | NIT
  4  6,2  | RAN
  4  6,5  | RAN
  4  5,4  | RASE
  4  6,5  | RATA
  4  6,5  | RATE
  4  5,1  - RE
  4  4,5  - RE
  4  7,1  | REA
  4  7,1  | REE
  4  7,4  | REIT
  4  6,2  | REN
  4  5,4  | REST
  4  7,1  | RET
  4  7,5  | RIN
  4  7,5  | RITA
  4  7,5  | RITE
  4  5,6  | SAN
  4  5,6  | SEN
  4  5,1  - TA
  4  4,5  - TA
  4  6,4  | TAAR
  4  6,5  | TAAR
  4  6,5  | TAN
  4  6,5  | TARA
  4  6,5  | TARE
  4  5,1  - TE
  4  4,5  - TE
  4  7,1  | TEA
  4  6,4  | TEAR
  4  7,1  | TEE
  4  7,5  | TIAR
  4  7,5  | TIER
  4  7,5  | TIN
  4  7,5  | TIRE
  4  5,5  | TSAR
  4  8,6  | YA
  4  8,6  | YE
  3  6,6  | AER
  3  5,4  | AES
  3  6,6  | AET
  3  7,5  | AIR
  3  7,5  | AIT
  3  6,6  | AN
  3  6,4  | ARA
  3  6,6  | ARA
  3  6,1  | ARE
  3  6,6  | ARE
  3  6,1  | ARR
  3  5,4  | ARS
  3  6,1  | ART
  3  6,6  | ART
  3  8,1  | ASE
  3  5,5  | ASE
  3  8,1  | AST
  3  5,5  | AST
  3  6,6  | ATE
  3  7,4  | ATI
  3  6,5  | EAR
  3  6,5  | EAT
  3  7,2  | EE
  3  7,5  | EIR
  3  7,2  | ER
  3  6,1  | ERA
  3  6,4  | ERA
  3  6,1  | ERR
  3  5,4  | ERS
  3  8,1  | EST
  3  5,5  | EST
  3  7,2  | ET
  3  6,4  | ETA
  3  7,6  | IN
  3  7,6  | IRA
  3  7,6  | IRE
  3  7,6  | ITA
  3 10,1  | NA
  3  6,5  | NA
  3  7,1  | NE
  3  9,2  | OE
  3  9,2  | OR
  3  5,4  | RAS
  3  6,2  | RAT
  3  6,5  | RAT
  3  6,2  | REA
  3  6,4  | REA
  3  5,4  | RES
  3  6,2  | RET
  3  7,5  | RIA
  3  7,5  | RIE
  3  7,5  | RIT
  3  6,2  | RTE
  3  7,4  | RTI
  3  5,6  | SAE
  3  5,6  | SAR
  3  5,6  | SAT
  3  5,6  | SEA
  3  5,6  | SER
  3  5,6  | SET
  3  5,6  | STA
  3  5,6  | STR
  3  6,4  | TAA
  3  6,5  | TAA
  3  6,5  | TAE
  3  7,4  | TAI
  3  6,5  | TAR
  3  5,4  | TAS
  3  6,4  | TEA
  3  7,5  | TIE
  3  6,1  | TRA
  3  6,4  | TRA
  3  7,4  | TRI
  3  5,4  | TRS
  2 10,1  | AA
  2  6,5  | AA
  2  6,6  | AA
  2  7,1  | AE
  2  6,6  | AE
  2  7,5  | AI
  2  6,1  | AR
  2  6,6  | AR
  2  8,1  | AS
  2  5,5  | AS
  2  6,6  | AT
  2  7,1  | EE
  2  6,1  | ER
  2  8,1  | ES
  2  5,5  | ES
  2  7,6  | IT
  2  7,1  | RE
  2  6,2  | RE
  2 10,1  | TA
  2  6,5  | TA
  2  7,1  | TE
  2  7,5  | TI
