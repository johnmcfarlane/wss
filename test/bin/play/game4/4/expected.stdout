6,0:RANT=30
6,0:RENT=30
7,0:ANT=25
6,0:RAN=25
6,0:NET=23
7,0:AN=20
1,0:ANTRE=20
7,0:EN=20
8,0:NA=20
7,0:ETA=19
8,4:QINTAR=19
6,0:RAT=19
6,0:RET=19
8,0:EON=18
8,0:TON=18
2,0:ANTE=17
2,0:ETNA=17
8,0:NOR=15
8,0:NOT=15
7,0:AT=14
7,0:ET=14
2,0:RATE=14
8,0:TA=14
2,0:TARE=14
4,2:TEN=14
5,6:ANE=13
5,5:ANEAR=12
5,5:ANTAE=12
5,5:ANTRA=12
5,5:ANTRE=12
5,5:ARENA=12
5,2:EN=12
4,2:NET=12
3,4:RANT=12
3,4:RENT=12
8,0:ROE=12
8,0:ROT=12
8,0:TOE=12
8,0:TOR=12
7,5:YENTA=12
5,0:ANE=11
4,4:ANT=11
1,2:ANTRE=11
5,6:ATE=11
5,6:ETA=11
2,4:NEAT=11
3,4:NEAT=11
9,3:NIDATE=11
8,2:EAU=10
6,6:NA=10
6,6:NE=10
4,2:RET=10
9,3:RIDENT=10
4,0:TARE=10
7,5:YEARN=10
7,5:YEN=10
2,2:ANTE=9
1,4:ANTE=9
0,4:ANTRE=9
5,0:ARE=9
4,4:EAT=9
5,0:ERA=9
4,4:ETA=9
3,0:NEAR=9
9,3:NIDE=9
8,0:NO=9
4,0:RAN=9
3,4:RAN=9
4,0:TAN=9
3,4:TAN=9
4,0:TEN=9
3,4:TEN=9
7,5:YARN=9
7,5:YEAN=9
9,3:AIDER=8
5,0:AN=8
4,4:AN=8
5,6:AN=8
3,2:ANE=8
4,4:ASTERN=8
5,0:EN=8
4,4:EN=8
5,6:EN=8
5,2:ER=8
5,2:ET=8
1,4:ETNA=8
6,0:NA=8
5,4:NA=8
2,4:NAE=8
6,0:NE=8
3,4:NET=8
9,0:RAN=8
2,2:RATE=8
7,1:SEN=8
4,5:STANE=8
4,5:STERNA=8
6,6:TA=8
9,0:TAN=8
2,2:TARE=8
3,0:TEAR=8
7,5:YARE=8
7,5:YEAR=8
9,3:AIDE=7
9,1:AN=7
2,4:ANE=7
4,0:EAR=7
3,4:EAT=7
6,5:INERT=7
6,5:INTER=7
3,2:NAE=7
4,2:NE=7
4,2:NETS=7
6,4:NITER=7
6,4:NITRE=7
9,2:NU=7
3,4:RAT=7
1,4:RATE=7
6,0:RENT=7
3,4:RET=7
6,4:RIANT=7
9,3:RIDE=7
4,5:SANER=7
4,5:SNARE=7
4,5:STERN=7
4,0:TAR=7
1,4:TARE=7
4,0:TEA=7
3,4:TEA=7
6,4:TINEA=7
7,5:YAR=7
7,5:YEA=7
7,5:YET=7
6,0:AE=6
9,3:AE=6
9,3:AID=6
3,0:ANE=6
4,2:ANES=6
4,2:ANTS=6
5,0:AR=6
3,2:ARE=6
2,4:ARE=6
4,4:ASTER=6
4,4:AT=6
5,4:AT=6
5,6:AT=6
3,2:ATE=6
2,4:ATE=6
5,0:ER=6
2,4:ERA=6
4,2:ERNS=6
5,0:ERRATA=6
4,4:ET=6
5,6:ET=6
2,4:ETA=6
6,5:IRATE=6
3,0:NAE=6
6,0:NEAR=6
6,0:NEAT=6
5,4:RATAN=6
6,0:RE=6
9,3:RE=6
9,3:RID=6
4,5:SANE=6
4,5:SENT=6
4,5:STARE=6
5,4:TA=6
3,2:TAE=6
2,4:TAE=6
4,2:TANS=6
2,4:TEA=6
6,0:TEEN=6
6,0:TEN=6
4,2:TENS=6
6,0:TERN=6
8,0:TO=6
6,3:TRINE=6
7,0:TSAR=6
4,2:AE=5
6,4:AIRN=5
5,5:ANTA=5
5,5:ANTE=5
3,0:ARE=5
4,2:ARES=5
4,2:ARTS=5
3,0:ATE=5
4,2:ATES=5
5,4:EARN=5
4,2:EARS=5
4,2:EATS=5
6,1:EN=5
3,0:ERA=5
4,2:ERAS=5
6,1:ERN=5
3,0:ETA=5
4,2:ETAS=5
9,4:IDEA=5
4,0:NA=5
3,4:NA=5
4,0:NE=5
3,4:NE=5
5,3:NEAR=5
5,3:NEAT=5
6,0:NEE=5
4,3:NEST=5
6,0:NET=5
6,4:NITE=5
8,1:ON=5
6,3:RAIN=5
5,4:RANT=5
4,2:RATS=5
4,2:RE=5
5,1:REATA=5
5,3:REATA=5
6,3:REIN=5
4,3:RESAT=5
6,0:RETE=5
4,2:RETS=5
7,1:SAE=5
7,1:SAT=5
4,5:SATE=5
7,1:SEA=5
4,5:SEAR=5
4,5:SEAT=5
7,1:SER=5
4,5:SERA=5
7,1:SET=5
4,5:SETA=5
4,5:STAR=5
3,0:TAE=5
6,3:TAIN=5
5,4:TARN=5
4,2:TARS=5
3,0:TEA=5
6,0:TEAR=5
4,2:TEAS=5
6,4:TINE=5
4,2:TRES=5
4,0:AE=4
3,4:AE=4
6,4:AIN=4
6,4:AIRT=4
5,3:ANA=4
5,5:ANA=4
5,5:ANE=4
6,3:ANI=4
5,5:ANT=4
5,5:AREA=4
4,3:ARSE=4
6,1:EAR=4
4,3:EAST=4
6,1:EAT=4
4,3:ENS=4
6,1:ERA=4
6,1:ERE=4
5,0:ERN=4
4,3:ERST=4
6,1:ETA=4
5,4:NAE=4
6,4:NIT=4
5,1:RAN=4
5,4:RAN=4
4,3:RASE=4
5,4:RATE=4
4,0:RE=4
3,4:RE=4
6,0:REE=4
4,3:REST=4
6,0:RET=4
6,4:RIN=4
6,4:RITE=4
4,5:SEN=4
4,0:TA=4
3,4:TA=4
5,4:TAN=4
5,4:TARE=4
6,0:TEA=4
5,3:TEAR=4
6,0:TEE=4
6,4:TIER=4
6,4:TIN=4
6,4:TIRE=4
4,4:TSAR=4
7,5:YA=4
7,5:YE=4
6,4:AIR=3
6,4:AIT=3
5,5:AN=3
5,0:ARE=3
5,5:ARE=3
4,3:ARS=3
5,0:ART=3
5,5:ART=3
5,5:ATE=3
5,4:EAR=3
5,4:EAT=3
6,1:ER=3
5,0:ERA=3
5,3:ERA=3
5,0:ERR=3
4,3:ERS=3
6,1:ET=3
5,3:ETA=3
6,5:IN=3
6,5:IRE=3
9,0:NA=3
5,4:NA=3
6,0:NE=3
8,1:OE=3
8,1:OR=3
6,3:RAI=3
4,3:RAS=3
5,1:RAT=3
5,4:RAT=3
6,3:REI=3
4,3:RES=3
5,1:RET=3
6,4:RIA=3
4,5:SAE=3
4,5:SAT=3
4,5:SEA=3
4,5:SER=3
4,5:SET=3
5,4:TAE=3
5,4:TAR=3
4,3:TAS=3
5,3:TEA=3
6,4:TIE=3
9,0:AA=2
5,4:AA=2
5,5:AA=2
6,0:AE=2
5,5:AE=2
6,4:AI=2
5,0:AR=2
5,5:AR=2
7,0:AS=2
4,4:AS=2
5,5:AT=2
5,0:ER=2
7,0:ES=2
4,4:ES=2
6,5:IT=2
6,0:RE=2
5,1:RE=2
9,0:TA=2
5,4:TA=2
6,4:TI=2
